src/subscriber.ts

```diff
@@ -771,11 +771,22 @@ export class Subscriber extends EventEmitter {
 
       const {maxStreams = defaultOptions.subscription.maxStreams} =
         options.streamingOptions;
+
       options.streamingOptions.maxStreams = Math.min(
         maxStreams,
         this.maxMessages
       );
     }
+
+    if (this._acks) {
```

Comment:
Let's not expose these settings more than they already are, since we don't want users to be able to set these batching settings. Flow control is ok though